baybe/_optional/onnx.py

```diff
@@ -3,7 +3,15 @@
 from baybe.exceptions import OptionalImportError
 
 try:
+    import onnx
     import onnxruntime
+
+    try:
+        from onnx.helper import split_complex_to_pairs  # noqa: F401
+    except ImportError:
+        from onnx.helper import _split_complex_to_pairs
```

Comment:
Can you please add the necessary context information here on why this is necessary? That is, mentioned the backward-incompatible change and the broken behavior of skl2onnx, ideally with a link to a corresponding issue you opened. Since otherwise, it's completely unclear why this is needed, especially since the error does not occur in our own code but in downstream dependencies.

Also, mention the breaking onnx version. In fact, you could even consider changing the `try/except` to an if-condition checking the version, which would make it very explicit.